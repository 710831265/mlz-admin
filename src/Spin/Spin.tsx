import React from 'react';
import { Spin as AntdSpin } from 'antd';

const Spin = AntdSpin;

export default Spin;
