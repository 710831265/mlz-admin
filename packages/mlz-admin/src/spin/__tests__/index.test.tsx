import React, { Component } from 'react';
import { testMount, testSnapshot } from '../../../../../tests';
import Spin from '..';

describe('ðŸ§ª Spin', () => {
  testMount(Spin);
  testSnapshot(Spin);
});
