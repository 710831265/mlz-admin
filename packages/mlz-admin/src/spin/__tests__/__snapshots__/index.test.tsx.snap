// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`🧪 Spin snapshot test 快照匹配 1`] = `"<div class=\\"ant-spin ant-spin-spinning\\"><span class=\\"ant-spin-dot ant-spin-dot-spin\\"><i class=\\"ant-spin-dot-item\\"></i><i class=\\"ant-spin-dot-item\\"></i><i class=\\"ant-spin-dot-item\\"></i><i class=\\"ant-spin-dot-item\\"></i></span></div>"`;
